apps/atomy-q/API/.env.example

```
@@ -126,6 +126,14 @@ AI_DOCUMENT_RETRY_ATTEMPTS=2
 AI_DOCUMENT_RETRY_BACKOFF_MS=500
 AI_DOCUMENT_MODEL_ID=
 AI_DOCUMENT_MODEL_REVISION=
+# Parser plugin for OpenRouter PDF/image document ingestion. Default `file-parser`.
+AI_DOCUMENT_PARSER_PLUGIN=file-parser
+# OCR engine for PDF parsing. Default `mistral-ocr`; update only when the provider supports another engine.
+AI_DOCUMENT_PDF_ENGINE=mistral-ocr
+# Max local document size loaded into provider payloads (bytes).
+AI_DOCUMENT_MAX_FILE_SIZE_BYTES=10485760
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AI_DOCUMENT_MAX_FILE_SIZE_BYTES key should go before the AI_DOCUMENT_MODEL_ID key

(UnorderedKey)